### PR TITLE
Add initial content to base template

### PR DESCRIPTION
**Pull request by AndresForondaPerf**

### CODE REVIEW

File: ui.content/src/main/content/jcr_root/conf/aem-bootcamp/settings/wcm/template-types/page/initial/.content.xml
Comment (lflondonol92) [inline ui.content/src/main/content/jcr_root/conf/aem-bootcamp/settings/wcm/template-types/page/initial/.content.xml:1]: This file is not part of the scope of the ticket

File: ui.content/src/main/content/jcr_root/conf/aem-bootcamp/settings/wcm/template-types/page/policies/.content.xml
Comment (lflondonol92) [inline ui.content/src/main/content/jcr_root/conf/aem-bootcamp/settings/wcm/template-types/page/policies/.content.xml:1]: This file is not part of the scope of the ticket

File: ui.content/src/main/content/jcr_root/conf/aem-bootcamp/settings/wcm/template-types/page/structure/.content.xml
Comment (lflondonol92) [inline ui.content/src/main/content/jcr_root/conf/aem-bootcamp/settings/wcm/template-types/page/structure/.content.xml:1]: This file is not part of the scope of the ticket

File: ui.content/src/main/content/jcr_root/conf/aem-bootcamp/settings/wcm/template-types/page/thumbnail.png.dir/.content.xml
Comment (lflondonol92) [inline ui.content/src/main/content/jcr_root/conf/aem-bootcamp/settings/wcm/template-types/page/thumbnail.png.dir/.content.xml:1]: This file is not part of the scope of the ticket

File: ui.content/src/main/content/jcr_root/conf/aem-bootcamp/settings/wcm/template-types/xf/.content.xml
Comment (lflondonol92) [inline ui.content/src/main/content/jcr_root/conf/aem-bootcamp/settings/wcm/template-types/xf/.content.xml:1]: This file is not part of the scope of the ticket

File: ui.content/src/main/content/jcr_root/conf/aem-bootcamp/settings/wcm/template-types/xf/initial/.content.xml
Comment (lflondonol92) [inline ui.content/src/main/content/jcr_root/conf/aem-bootcamp/settings/wcm/template-types/xf/initial/.content.xml:1]: This file is not part of the scope of the ticket

File: ui.content/src/main/content/jcr_root/conf/aem-bootcamp/settings/wcm/template-types/xf/policies/.content.xml
Comment (lflondonol92) [inline ui.content/src/main/content/jcr_root/conf/aem-bootcamp/settings/wcm/template-types/xf/policies/.content.xml:1]: This file is not part of the scope of the ticket

File: ui.content/src/main/content/jcr_root/conf/aem-bootcamp/settings/wcm/template-types/xf/structure/.content.xml
Comment (lflondonol92) [inline ui.content/src/main/content/jcr_root/conf/aem-bootcamp/settings/wcm/template-types/xf/structure/.content.xml:1]: This file is not part of the scope of the ticket

File: ui.content/src/main/content/jcr_root/conf/aem-bootcamp/settings/wcm/template-types/page/.content.xml
Comment (lflondonol92) [inline ui.content/src/main/content/jcr_root/conf/aem-bootcamp/settings/wcm/template-types/page/.content.xml:1]: This file is not part of the scope of the ticket

**Review by lflondonol92 — COMMENTED** [review]
Remove the unnecessary files.

File: ui.content/src/main/content/jcr_root/conf/aem-bootcamp/settings/wcm/template-types/page/initial/.content.xml
Comment (AndresForondaPerf) [reply ui.content/src/main/content/jcr_root/conf/aem-bootcamp/settings/wcm/template-types/page/initial/.content.xml:1]: Deleted

File: ui.content/src/main/content/jcr_root/conf/aem-bootcamp/settings/wcm/template-types/page/policies/.content.xml
Comment (AndresForondaPerf) [reply ui.content/src/main/content/jcr_root/conf/aem-bootcamp/settings/wcm/template-types/page/policies/.content.xml:1]: Deleted

File: ui.content/src/main/content/jcr_root/conf/aem-bootcamp/settings/wcm/template-types/page/structure/.content.xml
Comment (AndresForondaPerf) [reply ui.content/src/main/content/jcr_root/conf/aem-bootcamp/settings/wcm/template-types/page/structure/.content.xml:1]: Deleted

File: ui.content/src/main/content/jcr_root/conf/aem-bootcamp/settings/wcm/template-types/xf/.content.xml
Comment (AndresForondaPerf) [reply ui.content/src/main/content/jcr_root/conf/aem-bootcamp/settings/wcm/template-types/xf/.content.xml:1]: Deleted

File: ui.content/src/main/content/jcr_root/conf/aem-bootcamp/settings/wcm/template-types/xf/initial/.content.xml
Comment (AndresForondaPerf) [reply ui.content/src/main/content/jcr_root/conf/aem-bootcamp/settings/wcm/template-types/xf/initial/.content.xml:1]: Deleted

File: ui.content/src/main/content/jcr_root/conf/aem-bootcamp/settings/wcm/template-types/xf/structure/.content.xml
Comment (AndresForondaPerf) [reply ui.content/src/main/content/jcr_root/conf/aem-bootcamp/settings/wcm/template-types/xf/structure/.content.xml:1]: Deleted

File: ui.content/src/main/content/jcr_root/conf/aem-bootcamp/settings/wcm/template-types/page/.content.xml
Comment (AndresForondaPerf) [reply ui.content/src/main/content/jcr_root/conf/aem-bootcamp/settings/wcm/template-types/page/.content.xml:1]: Deleted

File: ui.content/src/main/content/jcr_root/conf/aem-bootcamp/settings/wcm/template-types/xf/policies/.content.xml
Comment (AndresForondaPerf) [reply ui.content/src/main/content/jcr_root/conf/aem-bootcamp/settings/wcm/template-types/xf/policies/.content.xml:1]: Deleted

File: ui.content/src/main/content/jcr_root/conf/aem-bootcamp/settings/wcm/template-types/page/thumbnail.png.dir/.content.xml
Comment (AndresForondaPerf) [reply ui.content/src/main/content/jcr_root/conf/aem-bootcamp/settings/wcm/template-types/page/thumbnail.png.dir/.content.xml:1]: Deleted

**Review by lflondonol92 — APPROVED** [review]
Approved

**Pull request merged**